tests/units/common/parsing/test_parser/input.json

```diff
@@ -42,7 +42,7 @@
   ],
   "collections": ["Test Collection"],
   "control_field": ["Test control field"],
-  "free_keywords": ["Test free 1", "Test free 2"],
+  "free_keywords": ["Test free 1", "Test free  2   "],
```

Comment:
why we need the space?

Reply:
yes, I am testing if are keywords are cleaned. Because these tests didn't test the cleaning part of keywords extraction at all